modules/ROOT/pages/reliability-patterns.adoc

```diff
@@ -0,0 +1,162 @@
+= Reliability Patterns
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: reliability, patterns, cluster, ha, transaction, delivery, ensure
+
+== Overview
+
+High-reliability applications must have zero tolerance for message loss. This means that both the underlying Mule and its individual connections need to be reliable. If your application uses a transactional connection such as JMS, VM, or DB, reliable messaging is ensured by the built-in support for transactions in the connector. This means, for example, that you can configure a transaction on a JMS listener that makes sure messages are only removed from the JMS server when the transaction is committed. By doing this, you ensure that if an error occurs while processing the message, it will still be available for reprocessing.
+
+In other words, the transactional support in these connectors ensures that messages are delivered reliably from a source to an operation or between processors within a flow. Note though that if you want to move messages between different connectors that support transactions you need to use XA transactions to ensure that both transports' transactions are committed as one atomic unit.
+
+//See xref:transaction-management.adoc[Transaction Management] for more information on XA and other types of transactions.
+
+If you have a web application that uses a non-transactional connector, such as HTTP, you must follow a reliability pattern to ensure reliable messaging for your application.
+
+[NOTE]
+Note that VM file persistency is disabled on clusters, so VM endpoints persist in-memory in a clustered topology.
+
+== Reliability Patterns
+
+A reliability pattern is a design that results in reliable messaging for an application, even if the application receives messages from a non-transactional transport. A reliability pattern couples a reliable acquisition flow with an application logic flow, as shown in the following diagram:
+
+image::reliability-pattern.png[Reliability_Pattern]
+
+The reliable acquisition flow (the left-hand part of the diagram) delivers a message reliably from a Message Source that does not implement transactions to an outbound operation of a connector that implements transactions. The operation can be of any type of the transactional endpoints, such as VM or JMS. If the reliable acquisition flow cannot deliver the message, it ensures that the message isn't lost:
```

Comment:
For translation, I think we aren't supposed to use the "hand"--so just say left part of the diagram. 
lc message source
Period at end of last sentence instead of :